Microsoft/WebTemplateStudio
# angular folder name should be kebab case

task:https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/WTS%20Team/Intern%20GitHub/Y19-VS/04

This PR is to achieve: **folder name should be kebab case and module name should be pascal case.** These two names were coupled with whatever the default name or user input name, also "-" will be converted into "_". Now with Sibille's change, I am able to get different casing names from dedault name or user input name. 

Sibille stored cased names in variable "Param_SourceName_Kebab" and "Param_SourceName_Parscal", so I just need to query them in template.json file and use it in the correct places, i.e. folder name, module name.

**How to test:**
1. generate a project with 4 types of page selected
2. the app should start properly
3. do "yarn lint" or "npm run lint", all files should pass

**Expected:**
![fodler-name](https://user-images.githubusercontent.com/12874317/59468710-99466500-8de7-11e9-91a3-52243a21c8ba.PNG)
